packages/components/src/components/VirtualizedList/VirtualizedList.tsx

```diff
@@ -44,13 +53,21 @@ type VirtualizedListProps<T> = {
     items: Array<T & { height: number }>;
     onScroll?: (e: Event) => void;
     onScrollEnd: () => void;
-    height: number | string;
+    listHeight: number | string;
+    listMinHeight: number | string;
     renderItem: (item: T, index: number) => React.ReactNode;
 };
 
 export const VirtualizedList = forwardRef<HTMLDivElement, VirtualizedListProps<any>>(
```

Comment:
1. The shadow at the bottom is over scrollbar
2. scrollbar should probably not be over items

https://github.com/user-attachments/assets/25da1bf5-4b11-4bdf-b8cb-da985d17d7d2

cc @jvaclavik